TableTopCampaignManager/DiscordBot.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Discord;
+using Discord.Net;
+using Discord.WebSocket;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Discordbot
+{
+    class DieRoller
+    {
+        public static void Main(string[] args)
+        => new DieRoller().InitializeClient().GetAwaiter().GetResult();
+        private DiscordSocketClient _client;
+        public async Task InitializeClient()
+        {
+            _client = new DiscordSocketClient();
+            _client.Ready += Client_Ready;
+            _client.Log += LogUserMessage;
+            var token = File.ReadAllText("token.txt");
+
+            await _client.LoginAsync(TokenType.Bot, token);
+            await _client.StartAsync();
+
+            await Task.Delay(-1);
+        }
+        private Task LogUserMessage(LogMessage msg)
+        {
+            Console.WriteLine(msg.ToString());
+            return Task.CompletedTask;
+        }
+        public async Task Client_Ready()
```

Comment:
Same as comment above

Reply:
Do you mean rename the Task called Client_Ready()? Since this Task is ultimately designed purely to initialize these commands, it should probably be named InitializeCommands() or RegisterCommands(). We'll have a different task for actually handling inputs from users.

Reply:
Actually, yes. That's a better suggestion